zeek-ocsf/package.yaml

```
@@ -1139,38 +1051,28 @@ pipelines:
       // === Context ===
       ocsf.metadata = {
         log_name: "x509.log",
-        logged_time: zeek._write_ts,
+        logged_time: zeek._write_ts?,
         product: {
           name: "Zeek",
           vendor_name: "Zeek",
           cpe_name: "cpe:2.3:a:zeek:zeek",
         },
         version: "1.4.0",
       }
-      drop zeek._path, zeek._write_ts
+      drop zeek._path? // implied in metadata.log_name
       ocsf.tls = {
         certificate: {
-          serial_number: zeek.certificate.serial,
-          created_time: zeek.certificate.not_valid_before,
-          expiration_time: zeek.certificate.not_valid_after,
-          issuer: zeek.certificate.issuer,
-          subject: zeek.certificate.subject,
-          fingerprints: [zeek.fingerprint],
-          version: zeek.certificate.version,
+          serial_number: move zeek.certificate.serial,
+          created_time: move zeek.certificate.not_valid_before,
+          expiration_time: move zeek.certificate.not_valid_after,
+          issuer: move zeek.certificate.issuer,
+          subject: move zeek.certificate.subject,
+          fingerprints: [move zeek.fingerprint?],
```

```suggestion
          fingerprints: [move zeek.fingerprint?].collect(),
```

This is a bit of a hack, but it makes it so that you get an empty list if the result of `move fingerprint?` is `null`.